src/client/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -66,6 +67,10 @@ class ErrorableTextInput extends React.Component {
       requiredSpan = <span className="hca-required-span">*</span>;
     }
 
+    if (this.props.charMax) {
+      maxCharacters = (<small>(Max. {this.props.charMax} characters)</small>);
```

Comment:
I think it might be better to only have this message appear if they reach 30 characters. And also perhaps have it be part of the error message.